### PR TITLE
[zh] sync glossary  kube-controller-manager

### PR DESCRIPTION
**Pull request by mengjiao-liu**
en PRs: https://github.com/kubernetes/website/pull/28957 https://github.com/kubernetes/website/pull/18438

[zh] sync glossary  kube-controller-manager file `content/en/docs/reference/glossary/kube-controller-manager.md`

### CODE REVIEW

File: content/zh/docs/reference/glossary/kube-controller-manager.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ tags:
 -->
 
 <!--
- Component on the master that runs {{< glossary_tooltip text="controllers" term_id="controller" >}}.
+ Control plane component that runs {{< glossary_tooltip text="controller" term_id="controller" >}} processes.
 -->
-在主节点上运行 {{< glossary_tooltip text="控制器" term_id="controller" >}} 的组件。
+运行 {{< glossary_tooltip text="控制器" term_id="controller" >}} 进程的控制平面组件。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kube-controller-manager.md:34]: 请删除{{之前和}}之后的空格

Comment (mengjiao-liu) [reply content/zh/docs/reference/glossary/kube-controller-manager.md]: Updated.

**Comment by tengqm** [comment]
Thanks
/lgtm
/approve

**Pull request merged**